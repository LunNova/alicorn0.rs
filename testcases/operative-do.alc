let foo = "baz"
let x = do
	let foo = "foo"
	let (foobar) = host-string-concat(foo, "bar")
	foobar
let (foobarbaz) = host-string-concat(x, foo)
foobarbaz
