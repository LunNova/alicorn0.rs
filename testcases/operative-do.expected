{ { "let", "foo", "=", { elements = { "baz" }, kind = "string" } }, { "let", "x", "=", "do", { "let", "foo", "=", { elements = { "foo" }, kind = "string" } }, { "let", { "foobar" }, "=", { "host-string-concat", "foo", { elements = { "bar" }, kind = "string" } } }, "foobar" }, { "let", { "foobarbaz" }, "=", { "host-string-concat", "x", "foo" } }, "foobarbaz" }
